packages/cli/package.json

```diff
@@ -83,6 +84,7 @@
     "@types/mkdirp": "^0.5.2",
     "@types/pretty-format": "^24.3.0",
     "@types/semver": "^6.0.2",
+    "@types/unzipper": "^0.10.2",
```

Comment:
this is unused now :)

Reply:
Really need to bring the script to check dependencies 🤦‍♂️